GNUmakefile

```diff
@@ -181,6 +183,7 @@ DO_TEST_TARGET := no
 ifeq (yes,$(ENABLE_FVTEST_AGENT))
 DO_TEST_TARGET := yes
 test_targets := $(test_prereqs)
+main_targets += omrtrace
```

Comment:
again why is this being added again?

Reply:
Similar answer to the above. fvtests require omrtrace. The duplicated entries get removed anyway a few lines below.